lorawan-device/Cargo.toml

```diff
@@ -27,21 +26,35 @@ defmt = { version = "0.3", optional = true }
 fastrand = { version = "2", default-features = false }
 futures = { version = "0.3", default-features = false }
 rand_core = { version = "0.6", default-features = false }
-serde = { version = "1", default-features = false, features = ["derive"], optional = true }
+serde = { version = "1", default-features = false, features = [
+    "derive",
+], optional = true }
 seq-macro = "0.3.5"
 document-features = "0.2.8"
 embassy-time = { version = "0.3.0", optional = true }
 
 [dev-dependencies]
-tokio = { version = "1", features = ["rt", "macros", "time", "sync"]}
+tokio = { version = "1", features = ["rt", "macros", "time", "sync"] }
 rand = { version = "0", features = ["getrandom"] }
 lazy_static = "1"
 # Pull in lorawan/default-crypto which is required for tests
-lorawan = { path = "../lorawan-encoding", features = ["default-crypto"] }
+lorawan = { path = "../lorawan-encoding", default-features = false, features = [
```

Comment:
Can you check whether running `cargo test --all-features` works from lora-phy after this change?